01_fib/cuminandpaprika/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fib(7)
+}
+
+// Print n number of fibonnaci sequence
+// 1, 1, 2, 3, 5, 8, 13
+// Note: we take the absolute value of n to ensure our loop works.
+func fib(n int16) {
+	var i int16
+	if isPositive(n) {
+		for ; i < n; i++ {
```

Comment:
good work unwrapping your loop 👍 

(I know premature optimisation is supposed to be evil, but I appreciate intuitive optimisations like this)

Reply:
Thanks!